src_web/kamaho-shokusu/tests/TestCase/Service/RoomUsageServiceTest.php

```
@@ -0,0 +1,121 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Test\TestCase\Service;
+
+use App\Service\RoomUsageService;
+use Cake\TestSuite\TestCase;
+
+class RoomUsageServiceTest extends TestCase
```

PHPStan (2.2.2)

[error] Class App\Test\TestCase\Service\RoomUsageServiceTest extends unknown class Cake\TestSuite\TestCase.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)